modules/apps/content-dashboard/content-dashboard-api/src/main/java/com/liferay/content/dashboard/item/action/ContentDashboardItemActionProvider.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.content.dashboard.item.action;
+
+import java.util.Locale;
+import java.util.Optional;
+
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * @author David Arques
+ */
+public interface ContentDashboardItemActionProvider {
```

Comment:
I won't add the ContentDashboardItemActionProvider to the API since this will never be used outside the web module